lib/Analysis/Allocation.cpp

```diff
@@ -135,10 +135,20 @@ getScratchConfigForCvtLayout(triton::gpu::ConvertLayoutOp op, unsigned &inVec,
   // TODO(jlebar): This is suboptimal if repShape[in/outOrd[0]] is small.  We
   // might be able to merge the few most-minor dimensions and get a larger
   // vector.
-  // For conversions to MmaV1 (Nvidia V100), this inVec is hardcoded in the
-  // codegen.
   inVec = std::min(srcContigPerThread[inOrd[0]], dstContigPerThread[inOrd[0]]);
   outVec = dstContigPerThread[outOrd[0]];
+
+  if (srcLayout.isa<BlockedEncodingAttr>() &&
+      dstLayout.isa<BlockedEncodingAttr>() && outOrd[0] != (rank - 1) &&
+      inOrd[0] != outOrd[0]) {
+    // Don't vectorize for transpose. Only the read or the write can be
+    // vectorized and this causes extra bank conflicts on the non-vectorized
+    // accesses.
```

Comment:
Not sure if you updated this?  The explanation you gave in the PR involved padding, which was important to my being able to understand.

Reply:
Oh, now there's an example.